lib/factiva/monitoring.rb

```diff
@@ -134,7 +134,7 @@ def create_association(first_name:, last_name:, birth_year:, external_id:, nin:,
 
     def add_association_to_case(case_id:, association_id:)
       params = { json: case_association_body(
-          association_id,
+        association_id,
```

Comment:
nitpick: This could be one line

Reply:
there was 45 offenses, I let rubocop to fix them himself